cmd/loadtest/recall.go

```diff
@@ -0,0 +1,77 @@
+package loadtest
+
+import (
+	"context"
+	"encoding/json"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/maticnetwork/polygon-cli/rpctypes"
+	"github.com/maticnetwork/polygon-cli/util"
+	"math/big"
+)
+
+// TODO allow this to be pre-specified with an input file
+func getRecentBlocks(ctx context.Context, ec *ethclient.Client, c *ethrpc.Client) ([]*json.RawMessage, error) {
+	bn, err := ec.BlockNumber(ctx)
+	if err != nil {
+		return nil, err
+	}
+	rawBlocks, err := util.GetBlockRange(ctx, bn-*inputLoadTestParams.RecallLength, bn, c)
+	return rawBlocks, err
+}
+
+func getRecallTransactions(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Client) ([]rpctypes.PolyTransaction, error) {
```

Comment:
so this fxn just returns all of the transactions of a given block range

Reply:
Yeah ideally. The goal being to replay this transactions.